TGM/src/main/java/network/warzone/tgm/util/menu/KitEditorMenu.java

```diff
@@ -175,13 +175,4 @@ public void onDrag(InventoryDragEvent event) {
             }
         }
     }
-
-    @EventHandler(priority = EventPriority.HIGHEST)
-    public void onClose(InventoryCloseEvent event) {
-        if (!event.getInventory().equals(inventory)) return;
-
-        Bukkit.getScheduler().runTask(TGM.get(), () -> {
-            event.getPlayer().getInventory().clear(0);
-        });
-    }
```

Comment:
This will make a visual bug possible, where an item which is on the cursor when using the kit editor will get "teleported" into the spectator inventory, if the kit editor is closed using the escape key.

Instead, the slot 0 should be changed to whatever lowest slot is unused after this PR.